test/integration/ClientQueryTest.java

```diff
@@ -217,7 +217,7 @@ public void applicationTest() {
             GraqlDelete deleteQuery = Graql.match(var("m").isa("mating")).delete(var("m").isa("mating"));
             LOG.info("clientJavaE2E() - '" + deleteQuery + "'");
             tx.execute(deleteQuery);
-            List<ConceptMap> matings = tx.execute(Graql.match(var("m").isa("mating")).get()).get();
+            List<ConceptMap> matings = tx.execute(Graql.match(var("m").isa("mating")).get("m")).get();
```

Comment:
You can also just omit the `.get(...)` method call altogether in the new Graql, @vmax. `Graql.match(...)` will already return a `GraqlMatch` query. But I'll approve this for now and up to you if you wanna make the change. This is also fine.